presto-main/src/test/java/com/facebook/presto/sql/planner/TestRowExpressionRewriter.java

```diff
@@ -34,17 +36,24 @@
 import static com.facebook.presto.sql.planner.TestRowExpressionRewriter.NegationExpressionRewriter.rewrite;
 import static com.facebook.presto.sql.relational.Expressions.call;
 import static com.facebook.presto.sql.relational.Expressions.constant;
+import static com.facebook.presto.sql.relational.RowExpressionNodeInliner.replaceExpression;
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertTrue;
 
 public class TestRowExpressionRewriter
 {
-    @Test
-    public void testSimple()
+    private FunctionManager functionManager;
```

Comment:
Would look a little cleaner if you move this logic to first commit. But I don't have strong feelings about this.